### ISSUE TITLE
Wrong code in Chapter2-04

### ISSUE THREAD

**Issue by Jinkeycode** [nfmcclure/tensorflow_cookbook#109]
tensorflow_cookbook/02_TensorFlow_Way/04_Implementing_Loss_Functions/04_loss_functions.py
tensorflow_cookbook/02_TensorFlow_Way/04_Implementing_Loss_Functions/04_loss_functions.ipynb

Sigmoid Entropy Loss function is
```
 tf.nn.sigmoid_cross_entropy_with_logits(logits=x_val_input, labels=target_input)
```
but in your code is
```
xentropy_sigmoid_y_vals = tf.nn.softmax_cross_entropy_with_logits(logits=x_val_input, labels=target_input)
```
line 71 on .py file and the same place on .ipynb

**Comment by nfmcclure** [comment]
Hi @Jinkeycode , wow, that's quite a typo, sorry about that and the late reply.  I'm just triaging everything now and getting ready to go through and update all the code for a book v2.

I'll update this very soon and you should see a fix in the next few weeks. Thanks again!

**Comment by nfmcclure** [comment]
This should be fixed now.  Thanks for the notice!
